app/views/admin_shelters/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<% @shelters.each do |shelter| %>
+  <h3><%= shelter.name %></h3>
+<% end %>
+
+
+<h2> Shelters with Pending Applications </h2>
+
+<% @shelters.each do |shelter| %>
+  <% shelter.pets.each do |pet| %>
```

Comment:
See my comment [here](https://github.com/turingschool-examples/adopt-dont-shop-7/pull/92/files#r1320598681) about this.